src/main/java/com/example/java/maven/game2048/MessagePrinter.java

```diff
@@ -1,4 +1,18 @@
 package com.example.java.maven.game2048;
 
 public class MessagePrinter {
+
+    private MessagePrinter() {
```

Comment:
This class should be responsible for `printing` messages to player. In this class you should NOT have nothing game specific. You should be able to use methods of this class in many console programs.
You should have here probably 2 methods:
```java
public static void printMessage(String messageToPrint){}

public static void printError(String messageToPrint){}
```

Case of this class is that you don't have `System.out.println` and `System.error.println` around the whole program - so when you want change the behavior (migrate from console to something else) you will need only change this one class.

Reply:
Done.